docs/tutorial/sync-alert-rules-from-git.md

```diff
@@ -1,2 +1,228 @@
 # Sync alert rules from git
 
+In this tutorial you will learn how to make COS Lite automatically sync the alert rules of a git repository to Prometheus using the [COS Configuration charm](https://charmhub.io/cos-configuration-k8s/docs).
+
+
+## Prerequisites
+This tutorial assumes that you already have the following:
+- A Kubernetes deployment [bootstrapped with Juju](https://documentation.ubuntu.com/juju/latest/tutorial/).
+- A git repository with your Prometheus Alert rules.
+
+## Introduction
+
+Imagine your business is looking to migrate your observability solution to COS Lite, but that you already have a lot of time invested into converting operational knowledge into alert rules that you're versioning in a git repository.
```

Comment:
```suggestion
Imagine your business wants to migrate its observability solution to COS-Lite, but you already have a lot of time invested into converting operational knowledge into alert rules that you're versioning in a git repository.
```

Reply:
I'll skip this one. Thanks for the suggestion! 🙇🏼